# Part pose
geometry_msgs/Pose part_pose
---

# Number of solutions returned
int8 num_sols
ik_service/JointSolutions[8] joint_solutions
